api/filters/namespace/example_test.go

```diff
@@ -40,11 +40,11 @@ metadata:
 	// kind: Foo
 	// metadata:
 	//   name: instance
-	//   namespace: app
+	//   namespace: "01234"
```

Comment:
same here, rather than changing the existing test case, could you add a test to `namespace_test.go`?

Reply:
Sounds good, I already added one (farther down in the pr) and will remove this as well.